proto/orchestrator.proto

```diff
@@ -51,14 +52,16 @@ service Orchestrator {
       returns (ListAssessmentToolsResponse) {
     option (google.api.http) = {
       get : "/v1/orchestrator/assessment_tools"
-      additional_bindings {get : "/v1/orchestator/{metric_id}/assessment_tools"}
+      additional_bindings {get : "/v1/orchestrator/{metric_id}/assessment_tools"}
+      response_body : "tools"
```

Comment:
Yes, you cannot use repeated fields.

Reply:
Yes, you cannot use repeated fields. But in this case a * should be sufficient, right?

Reply:
My goal was to get sth. like
`{tools: {id: "tool1",...}, {id: "tool2", ...},...}` 
instead of
`{ListAssessmentToolsResponse : tools: [{id: "tool1",...}, {id: "tool2", ...},...]}`

Looking in the openapi folder of the main branch, though, it seems that it automatically does the former one when `response_body` is omitted (being equal to `*`). I may do some tests with Postman, but I guess it is done right.